src/main/scala/com/linagora/openpaas/gatling/Configuration.scala

```diff
@@ -51,6 +51,8 @@ object Configuration {
   val InjectDuration = Properties.envOrElse("INJECT_DURATION", "10").toInt seconds
   val ScenarioDuration = Properties.envOrElse("SCENARIO_DURATION", "10").toInt seconds
   val UserCount = Properties.envOrElse("USER_COUNT", "1").toInt
+  val CustomRampUserCount = if (System.getProperty("rampUserCount") != null) Integer.parseInt(System.getProperty("rampUserCount")) else 20
```

Comment:
nitpicking but you could use the 'Properties.envOrElse' method like in the other configuration entries

Reply:
This one is not an environment variable that can be put in `env.sh` like the others. It is a SBT's system property, and cannot be retrieved via `Properties.envOrElse`.

Reply:
ah ok, i didn't see it was a  Property